### PR TITLE
added url function and its test

### PR DESCRIPTION
**Pull request by danderu**
@vincepal @FranDelaCasa @carlosvillu @davecarter please review!

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -32,6 +32,12 @@ export default class Rosetta extends EventEmitter {
     return this.translator.translate(key, values);
   }
 
+  url(urlPattern) {
```
Comment (carlosvillu) [inline src/i18n.js]: What happen with the pattern:

bcn/tag-tag-tag/moretag/index.asp

**Comment by vincepal** [comment]
:+1:

File: src/i18n.js (hunk 1 above)
Comment (danderu) [inline src/i18n.js]: We only have 3 or 4 cases of the `tag-tag-tag` pattern in the home page, and they are very static and specific cases. I'm planning to add them as resources. When we move to decouple the results and detail pages, we'll have a new routing sistem and we'll have to solve this problem globally.

**Comment by carlosvillu** [comment]
:+1:

**Pull request merged**